client/admin.lua

```diff
@@ -499,6 +499,24 @@ RegisterNetEvent('qbx_admin:client:Show', function()
     end
 end)
 
+lib.callback.register('qbx_admin:client:SaveCarDialog', function()
+    local response = lib.alertDialog({
+        header = 'Save Car',
+        content = 'This vehicle is already owned, do you want to override the current owner?',
+        centered = true,
+        cancel = true,
+        labels = {
+            confirm = 'Yes',
+            cancel = 'No'
+        }
+    })
+    return response == 'confirm' and true or false
+end)
+
+lib.callback.register('qbx_admin:client:GetVehicleInfo', function()
+    return GetDisplayNameFromVehicleModel(GetEntityModel(vehicle)):lower(), lib.getVehicleProperties(cache.vehicle)
```

Comment:
```suggestion
    return GetDisplayNameFromVehicleModel(GetEntityModel(cache.vehicle)):lower(), lib.getVehicleProperties(cache.vehicle)
```
My bad missed this one